src/components/EStellaAutocomplete.js

```diff
@@ -2,7 +2,7 @@ import PropTypes from 'prop-types';
 import {Autocomplete} from "@material-ui/lab";
 import {TextField} from "@material-ui/core";
 
-export const AutocompleteFilter = (props) => {
+export const EStellaAutocomplete = (props) => {
```

Comment:
Czemu EStellaAutocomplete a nie EStellaAutoCompleteFilter?

Reply:
Żebym ja wiedział skąd to się znalazło w tym PRze xDDD